README.md

```diff
@@ -186,12 +186,15 @@ Therefore the overall architecture will consist of the following elements:
     -   Used as a data sink to hold time-based historical context data
     -   offers an HTTP endpoint to interpret time-based data queries
 
--   A **Context Provider**: - A webserver acting as set of
-    [dummy IoT devices](https://github.com/Fiware/tutorials.IoT-Sensors) using
-    the
-    [Ultralight 2.0](https://fiware-iotagent-ul.readthedocs.io/en/latest/usermanual/index.html#user-programmers-manual)
-    protocol running over HTTP. - Note the **Stock Management Frontend** and
-    **Context Provider NGSI** proxy are not used in this tutorial.
+-   A **Context Provider**:
+
+    -   A webserver acting as set of
+        [dummy IoT devices](https://github.com/Fiware/tutorials.IoT-Sensors) using
+        the
+        [Ultralight 2.0](https://fiware-iotagent-ul.readthedocs.io/en/latest/usermanual/index.html#user-programmers-manual)
+        protocol running over HTTP. 
+    -   Note the **Stock Management Frontend** and
```

Comment:
The original was correct - actually is just a hyphen, not a bullet point - please revert.